File: semestru 4/MotoProject/Networking/src/main/java/network/dto/UserDTO.java
package network.dto;

import java.io.Serializable;


public class UserDTO implements Serializable{
    private String username;
    private String passwd;

    public UserDTO(String id) {
        this(id,"");
    }

    public UserDTO(String id, String passwd) {
        this.username = id;
        this.passwd = passwd;
    }

    public String getUsername() {
        return username;
    }

    public void setUsername(String id) {
        this.username = id;
    }

    public String getPasswd() {
        return passwd;
    }

    @Override
    public String toString(){
        return "UserDTO["+username+' '+passwd+"]";
    }
}

File: semestru 4/MotoProjectGrpcCs/networking/dto/UserDTO.cs
namespace moto.networking.dto;

[Serializable]
public class UserDTO
{
    public string Username { get; set; }
    public string Passwd { get; set; }
    
    public UserDTO(){}
    
    public UserDTO(string id) : this(id,"")
    {
    }
    public UserDTO(string id, string passwd)
    {
        Username = id;
        Passwd = passwd;
    }
}